src/components/Onboarding/Screens/GardenMetadata.js

```diff
@@ -1,22 +1,566 @@
-import React from 'react'
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import {
+  Box,
+  Button,
+  Field,
+  GU,
+  Help,
+  IconPlus,
+  IconTrash,
+  TextInput,
+  textStyle,
+  useTheme,
+} from '@1hive/1hive-ui'
 import { useOnboardingState } from '@providers/Onboarding'
+import ImageUploader from '../../ImageUploader'
 import Navigation from '../Navigation'
+import { publishNewDao } from '../../../services/github'
 
-function GardenMetadata({ title }) {
-  const { onBack, onNext, step, steps } = useOnboardingState()
+const COMMUNITY_LINK_TYPE = 'community'
+const DOCUMENTATION_LINK_TYPE = 'documentation'
+
+const DAO_LOGO = 'logo_type'
+const DAO_LOGO_MOBILE = 'logo'
```

Comment:
I wouldn't call this logo_mobile since it is present in desktop screens as well. 

Let's maybe stick with what we already use `logo` and `logo_type`

If you agree with this, let's also update the function namings
```suggestion
const DAO_LOGO_TYPE = 'logo_type'
const DAO_LOGO = 'logo'
```